src/shared/environment.ts

```diff
@@ -10,3 +10,8 @@ export const IS_TEST_ENV = process.env.TEST_ENV === 'true';
 export const SEGMENT_WRITE_KEY = process.env.SEGMENT_WRITE_KEY ?? '';
 export const SENTRY_DSN = process.env.SENTRY_DSN ?? '';
 export const TRANSAK_API_KEY = process.env.TRANSAK_API_KEY ?? '';
+
+// Feature flags
+export const LEDGER_ENABLED = true;
+export const SEGMENT_ENABLED = true;
+export const SENTRY_ENABLED = false;
```

Comment:
Can we instead infer segment/sentry enabled by presence of key?

I don't really feel that feature flags belong here, even though sometimes they may be toggled by environment. It's a slightly different concern

Reply:
I thought the point of this was to make it really explicit, but yeah, I can just use the SENTRY_DSN instead.

Reply:
Can we make a separate file for `feature-flags` bc I find them lost in the shared `constants` file. In researching moving the env variables, it was suggested that feature flags be in the .env file so that is why I thought to move them here.

Reply:
Happy with a separate feature flags file. Honestly, I'd just make a single object export rather than loads of individual const exports.

Disabling sentry is indeed something we want to make sure is disabled currently, exceptional case (make it false and add a comment), but segment lets say, having both "integration flag" and an env var makes for an odd matrix: 

1. flag on, no key
2. flag off, no key
3. flag on, key set
4. flag off, key set

it's confusing to have all possible options when only `#3` works